pages/contacts/[id]/index.js

```diff
@@ -1,8 +1,9 @@
 import styled from "styled-components";
 import { useRouter } from "next/router";
-import { contacts } from "../../../lib/db";
-import { StyledImage } from "../../../components/ContactListItem";
+// import { contacts } from "../../../lib/db";
```

Comment:
As we do not want outcommented code, this line should be removed. :)

Reply:
removed